src/controls/TouchControls.cpp

```diff
@@ -88,15 +145,19 @@ UIAction TouchControls::get_action()
     if (event.eventX >= 10 && event.eventX <= 10 + ui_button_width && event.eventY < 100)
     {
       action = DOWN;
+      renderPressedState(renderer, UP, false);
```

Comment:
@cgreening is not possible in this class to be aware of the mode right (list or read)
Because in read mode we could save the false pressed state since it seems to refresh the top part too when moving to the next page.